recipes/rt-utils/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "rt-utils" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qurit/rt-utils/archive/refs/tags/v{{ version }}.zip
+  sha256: 547be04ddaa566c357bea802392576aa3adf4d19f9284e4120ff21ebc308ec6b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps . -vv
```

https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```